src/parsec_security/parsec_threat_model/threat_model.md

```diff
@@ -147,27 +147,27 @@ mitigation status and the assets impacted. A threat can be unmitigated (U), miti
-| I | An attacker steals the authentication token from a client's request to then execute any operation in their name.                                                              | M-3                              | AS4, AS3 |
-| D | An attacker modifies the valid request of another client to modify the operation and make it fail.                                                                            | M-3                              | AS6      |
-| D | An attacker overloads/crashes the system and thus prevents any other user from making use of the service.                                                                     | Crashes: M-0, M-5 Overloads: U-0 | AS6      |
-| E | A malicious request, formatted in a specific way, triggers remote code execution in Parsec privilege level.                                                                   | U-3, M-5                         | All      |
-| E | A malicious request exploits a vulnerability in the software stack and leads to an attacker having Parsec privilege level on key management in the underlying hardware.       | U-3, M-5                         | AS3, AS6 |
+|   | Description                                                                                                                                                                                 | Mitigation                       | Assets   |
+|---|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------|----------|
+| S | (GNRC) An attacker uses another client's authentication token. He can then have access (reference) to any of the impersonated client's keys and execute any operation in their name.        | U-2                              | AS3      |
+| S | (GNRC) An attacker uses brute-force on the AUTH field of the request until he does not have an AuthenticationError response status to then deduce a valid authentication token.             | M-2                              | AS1      |
+| S | (DSPC) An attacker uses another client's Unix account. He can then have access (reference) to any of the impersonated client's keys and execute any operation in their name.                | U-5                              | AS3      |
+| S | (DSPC) An attacker uses a Unix account with the same UID as a client. He can then have access (reference) to any of the impersonated client's keys and execute any operation in their name. | O-11, ASUM-3                     | AS3      |
+| S | An attacker uses a weakness or a bug of one authentication method as soon as the authentication method used by Parsec is changed.                                                           | O-12, U-6                        | AS3      |
```

Comment:
I'm not sure I understand this one - more precisely, I don't get what `as soon as the authentication method used by Parsec is changed` means.

Reply:
That is in the context of Parsec supporting only one authenticator. If you change the authenticator to a weaker one containing a security bug, then all of your keys are as secure (for spoofing) as this authenticator is. And that happens as soon as the authenticator is changed.